libmamba/data/Mamba.psm1

```diff
@@ -58,7 +58,7 @@ function Exit-MambaEnvironment {
         if ($deactivateCommand.Trim().Length -eq 0) {
             return;
         }
-        Write-Verbose "[micromamba shell deactivate -s powershell]`n$deactivateCommand";
+        Write-Verbose "[micromamba shell deactivate --shell powershell]`n$deactivateCommand";
```

Comment:
Is this change aiming to make the command more explicit?
If so, should we change it everywhere?

Reply:
Yhea, it's cosmetic. At first I started looking at them because I wanted to remove usages of  micromamba shell --shell <shell> <cmd>` in favor of  `micromamba shell <cmd> --shell <shell>`